Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py

```diff
@@ -729,14 +729,18 @@ def _poll_collection(self, poll_service, begin, end):
             data=req,
             stream=True
         )
+        result.raw.decode_content = True
 
         while True:
             result_part_number = None
             result_id = None
             more = None
             tag_stack = collections.deque()  # type: ignore
+            xml_parser = etree.iterparse(result.raw, events=('start', 'end'), recover=True, dtd_validation=False,
+                                         load_dtd=False, encoding='utf-8')
```

Comment:
Why did you remove this from the try block? 
`load_dtd` and `dtd_validation` are False by default, please remove (see: https://lxml.de/api/lxml.etree.iterparse-class.html).
Why did you add `encoding` for `utf-8`? What happens when you don't provide encoding?

Reply:
As I said, the cs provided this solution and I asked her about those exact additional args :)
I will update here when I have more information.
I will move the line inside the try block again